src/common/row_operations/row_match.cpp

```diff
@@ -47,13 +47,13 @@ idx_t SelectComparison<GreaterThanEquals>(Vector &left, Vector &right, const Sel
 template <>
 idx_t SelectComparison<LessThan>(Vector &left, Vector &right, const SelectionVector *sel, idx_t count,
                                  SelectionVector *true_sel, SelectionVector *false_sel) {
-	return VectorOperations::GreaterThanEquals(left, right, sel, count, true_sel, false_sel);
+	return VectorOperations::LessThan(left, right, sel, count, true_sel, false_sel);
```

Comment:
Whoops!

Reply:
The funny part is that I saw the same issue somewhere else a while and fixed it, but now it's my turn!